### ISSUE TITLE
MOT 17 Training 

### ISSUE THREAD

**Issue by sawhney-medha** [MCG-NJU/MeMOTR#11]
Can you please share details about the validation set you used to validate your method for the MOT17 dataset before submitting to the test server?

**Comment by HELLORPG** [comment]
There is a head-scratching thing behind this scene. As we discussed in our paper, MOT17 has too little training data (~5000 frames) to train our method, because the DETR-like structure needs enough data to learn robust capabilities. We found a severe overfitting issue during training, so we did not set train_half and val_half like previous methods, e.g. ByteTrack, OC-SORT.

Actually, due to the limitation of our resources, I did not have enough time to study the impact of specific configs on MOT17. After I verified the effectiveness of the idea on DanceTrack, I directly transferred to MOT17 to run the final experiment. In the beginning, I leverage the configuration from MOTR. However, after I observed the overfitting issue, I decided to use fewer training epochs and finally got the results reported in our paper. **Therefore, I have not actually conducted validation experiments on MOT17.** If you want to conduct validation experiments on MOT17, you may could consider referring to MOTR and using the MOT15 train set as the val set. No official validation set is really an inconvenient problem.
